pkg/cmd/cli/cmd/rsh.go

```diff
@@ -143,5 +144,15 @@ func (o *RshOptions) Validate() error {
 
 // Run starts a remote shell session on the server
 func (o *RshOptions) Run() error {
+	// Insert the TERM into the command to be run
+	originalCommand := o.Command
+	o.Command = []string{"/bin/sh", "-c", fmt.Sprintf("TERM=vt100 %s", strings.Join(o.Command, " "))}
+
+	if o.ExecOptions.Run() == nil {
+		return nil
+	}
+
+	// If it failed, try again the original command, without the TERM addition
+	o.Command = originalCommand
```

Comment:
This seems super sketchy. Any rsh session returning a failure is run twice? It also assumes all commands are ok to run twice?